easybuild/easyconfigs/s/STAR/STAR-2.7.10a-GCC-11.3.0.eb

```diff
@@ -0,0 +1,42 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on STAR-2.7.7a-GCC-10.2.0.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'STAR'
+version = '2.7.10a'
+
+homepage = 'https://github.com/alexdobin/STAR'
+description = "STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays."
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Comment:
@maxim-masterov please restore the `software_license` comment.

Reply:
@verdurin which one? didn't find it in the previous versions of the easyconfig.

Reply:
```
# software_license = 'LicenseMIT'
```

Reply:
@verdurin @akesandgren done